### DISCUSSION TITLE
各位大佬，proxy-groups 中，如何使用 filter 来实现包含 AB 或 CD，但是不包含 EF 的过滤？

### DISCUSSION THREAD

**Discussion by yesiamchinese** [MetaCubeX/mihomo#1608] [Q&A]
例如：
  - name: USA
    type: url-test
    url: 'http://www.gstatic.com/generate_204'
    interval: 600
    use:
      - all-socks5
    filter: "(AB|CD).!EF"

这样会导致 USA 组一个节点都没有。。。

请教这个 filter 应该怎样写呢？

**Comment by Kwisma** [comment]
这样写

```
filter: "(?i)AB|CD"
```
